config.ru

```diff
@@ -36,7 +36,10 @@ if ! in_development || ENV["API_CACHE"]
   )
   if File.exists? cache_config_file_path
     template = ERB.new(File.read(cache_config_file_path)).result
-    use Rack::Cache, YAML.load(template).symbolize_keys
+    cache_config = YAML.load_file(template).symbolize_keys
```

Comment:
YAML.load loads from a string, not a file. This was probably causing the cache config to fall back and continue using heap storage, not redis.